scripts/exregional_run_post.sh

```diff
@@ -277,15 +278,26 @@ mv_vrfy BGRD3D.GrbF${post_fhr} ${bgrd3d}
 # small subset of surface fields for testbed and internal use
 wgrib2 -match "APCP|parmcat=16 parm=196|PRATE" ${bgrd3d} -grib ${bgsfc}
 
-# Link output for transfer from Jet to web
+#Link output for transfer to Jet
+# Should the following be done only if on jet??
+
+# Seems like start_date is the same as "$yyyymmdd $hh", where yyyymmdd
+# and hh are calculated above, i.e. start_date is just cdate but with a
+# space inserted between the dd and hh.  If so, just use "$yyyymmdd $hh"
+# instead of calling sed.
 basetime=$( date +%y%j%H%M -d "${yyyymmdd} ${hh}" )
-ln_vrfy -fs ${bgdawp} ${postprd_dir}/BGDAWP_${basetime}${post_fhr}00
-ln_vrfy -fs ${bgrd3d} ${postprd_dir}/BGRD3D_${basetime}${post_fhr}00
-ln_vrfy -fs ${bgsfc} ${postprd_dir}/BGSFC_${basetime}${post_fhr}00
+cp_vrfy ${bgdawp} ${COMOUT}/${NET}.t${cyc}z.bgdawpf${fhr}.${tmmark}.grib2
+cp_vrfy ${bgrd3d} ${COMOUT}/${NET}.t${cyc}z.bgrd3df${fhr}.${tmmark}.grib2
+cp_vrfy ${bgsfc}  ${COMOUT}/${NET}.t${cyc}z.bgsfcf${fhr}.${tmmark}.grib2
```

Comment:
Consider finding a way to extract the file name from `bgdawp`, etc. instead of copying here. Duplication of information can make this difficult to support moving forward.

Reply:
an off topic question, what does "bgdawp" stand for?

Reply:
I don't know but the file includes pressure level, hybrid level, and surface fields. I guess it is for AWIPS.

Reply:
@guoqing-noaa: "bgd" is for B-grid from the NMMB, and I believe "awp" is for AWIPS.

Reply:
Thanks, Jeff and Ming!